common/display/displayplanemanager.cpp

```diff
@@ -111,6 +111,9 @@ std::tuple<bool, DisplayPlaneStateList> DisplayPlaneManager::ValidateLayers(
   auto layer_begin = layers.begin();
   auto layer_end = layers.end();
   bool render_layers = false;
+  // Below flag takes care of condition,when the cursor layer is assigned the
+  // lower zorder
+  bool cursor_plane_enable = true;
```

Comment:
@pyn1 Better to ensure the z-order is correct for cursor when populating layers in present call in drmhwctwo.cpp rather than complicating logic here.